features/sharding/core/src/test/java/org/apache/shardingsphere/sharding/merge/dql/ShardingDQLResultMergerTest.java

```diff
@@ -515,6 +515,16 @@ private ShardingSphereDatabase createDatabase() {
                 mock(RuleMetaData.class), Collections.singletonMap(DefaultDatabase.LOGIC_NAME, schema));
     }
     
+    private ShardingSphereDatabase createSQLServerDatabase() {
```

Comment:
Why is this method called createSQLServerDatabase, but MySQL is used internally?